packages/vuefire/test/options.spec.js

```diff
@@ -1,10 +1,44 @@
 import { firestorePlugin } from '../src'
-import { Vue } from '@posva/vuefire-test-helpers'
+import { db, delay, Vue } from '@posva/vuefire-test-helpers'
+
+const createLocalVue = () => {
+  const newVue = Vue.extend()
+  newVue.config = Vue.config
+  return newVue
+}
 
 describe('Firestore: plugin options', () => {
   it('allows customizing $rtdbBind', () => {
-    Vue.use(firestorePlugin, { bindName: '$myBind', unbindName: '$myUnbind' })
-    expect(typeof Vue.prototype.$myBind).toBe('function')
-    expect(typeof Vue.prototype.$myUnbind).toBe('function')
+    const LocalVue = createLocalVue()
+    LocalVue.use(firestorePlugin, { bindName: '$myBind', unbindName: '$myUnbind' })
+    expect(typeof LocalVue.prototype.$myBind).toBe('function')
+    expect(typeof LocalVue.prototype.$myUnbind).toBe('function')
+  })
+  it('allows global use of a custom createSnapshot function', async () => {
```

Comment:
empty line between tests, we could even add the https://github.com/jest-community/eslint-plugin-jest/, it should have a rule for that

Reply:
I couldn't find a specific rule for that, but I added the line